src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -335,6 +335,12 @@ public static bool IsSZArray(this TypeSymbol type)
             return type.TypeKind == TypeKind.Array && ((ArrayTypeSymbol)type).IsSZArray;
         }
 
+        public static bool IsFunctionPointerType(this TypeSymbol type)
+        {
+            RoslynDebug.Assert((object)type != null);
```

Comment:
>RoslynDebug.Assert((object)type != null); [](start = 12, length = 41)

It is not clear what is the purpose of the assert.

Reply:
I'm following the same conventions as the rest of the `IsX` above.

---
In reply to: [382385354](https://github.com/dotnet/roslyn/pull/41794#discussion_r382385354) [](ancestors = 382385354)